### PR TITLE
#U11 feat  project-session to the tablet

### PR DESCRIPTION
**Pull request by IbrahimBagalwa**
# PR Description

design the project section to the tablet, when we tap the image-project should be under the title.
# Description of tasks that were expected to be completed

List all the tasks and sub-tasks that were assigned to you  Make sure you check every completed task. list them in this style

- [x]  In this feature when you click to the Title you show the snapshot under the title.
- [x] it should fit to the iPad
- [x] it should fit to the iPad pro-11

# Description of tasks that were expected to be completed and that are not yet complited

- [x] Animations

# How this maybe tested?

Please describe the tests that you ran to verify your functionalities. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

1. run `npm i` to install all required dependences 
2. run `npm run dev`

# Please check this Checklist before you submit your PR:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My code generate no warnings
- [x] There are no vulnerabilities

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -49,13 +51,13 @@ module.exports = {
...
-        50:"50.375rem"
+        50: "50.375rem",
       },
       letterSpacing: {
         0.01: "0.01rem",
       },
       screens: {
-        ssm: "400px",
+        ssm: "25rem",
       },
```
Comment (Yvad60) [inline tailwind.config.js:62]: Good catch!

File: components/sections/Project.jsx [hunk 2]
```diff
@@ -8,14 +8,14 @@ export default function Project() {
...
+    <div className="flex flex-col mb-13 md:mb-23">
       <h4 className="text-xl text-muted-300">Projects</h4>
 
       {projectData.map((items) => {
         const { id, image, title, subtitle } = items;
 
         return (
-          <div key={id}>
+          <div key={id} className="">
             <h3
```
Comment (Yvad60) [inline components/sections/Project.jsx:19]: This empty class name is not necessary

**Review by Yvad60 — APPROVED** [review]
Nice! Looks good

File: tailwind.config.js (hunk 1 above)
Comment (serapieTuyishime) [reply tailwind.config.js:62]: yeah

**Review by serapieTuyishime — APPROVED** [review]

**Pull request merged**